src/utils/validation.js

```diff
@@ -8,6 +8,13 @@ const requiredMessage = 'This field is required';
 // Basic validation that ensures some value was entered
 const basicValidate = value => (value ? undefined : requiredMessage);
 
+// Validate a course tag inside ReduxFormCreatableSelect
+// Disallow already entered tags
+function courseTagValidate(tagValue, selectValue, options) {
+  if (!/^[\p{L}\p{N}_-]+$/u.test(tagValue)) { return false; }
```

Comment:
nit: Add a comment explaining what this regex is doing.
question: shouldn't tag always be in English? I am not sure if non-English tags are allowed at this moment.

Reply:
umm fixed it to allow only english characters....

before I was allowing unicode because in the discovery admin it said something like `Only unicode characters, numbers, hyphen and underscore allowed`